packages/core/admin/admin/src/pages/AuthPage/components/Register/index.js

```diff
@@ -271,7 +271,8 @@ const Register = ({ fieldsToDisable, noSignin, onSubmit, schema }) => {
                     })}
                     type={confirmPasswordShown ? 'text' : 'password'}
                   />
-                  <Checkbox
+                  {/* To uncomment in the next release */}
```

Comment:
Same here.

Reply:
Should we remove it since we will have to add it again in 2 weeks? I just didn't want us having to recode it exactly the same for such a temporary feature?